### PR TITLE
Virus scans and super-linter added

### PR DESCRIPTION
**Pull request by Mionsz**
Virus scans and super-linter added
- Cpp and C scans with clang style enabled.
- Python scanner enabled
- Golang scanner enabled.
- Markdown lint enabled.

### CODE REVIEW

**Review by ko80 — DISMISSED** [review]

File: .github/workflows/ubuntu-build.yml [hunk 1]
```diff
@@ -83,10 +83,29 @@ jobs:
...
-      run: ./build.sh
+      run: eval 'source scripts/common.sh && ./build.sh "${PREFIX_DIR}"'
 
     - name: 'Build MCM FFmpeg plugin'
       working-directory: ${{ github.workspace }}/ffmpeg-plugin
       run: |
         ./configure-ffmpeg.sh && \
         ./build-ffmpeg.sh
+
+    - name: 'Scaner: Send to VirusTotal for analysis'
```
Comment (MateuszGrabuszynski) [inline .github/workflows/ubuntu-build.yml:94]: [minor]
```suggestion
    - name: 'Scanner: Send to VirusTotal for analysis'
```

File: .github/workflows/ubuntu-build.yml [hunk 2]
```diff
@@ -83,10 +83,29 @@ jobs:
...
+            _install/usr/*
+            _install/usr/lib64/*
+            _install/usr/local/bin/*
+            _install/usr/local/lib/*
+            _install/usr/local/lib/x86_64-linux-gnu/*
+            _install/usr/local/lib/x86_64-linux-gnu/dpdk/*
+            _install/usr/local/lib/bpf/*
+            _install/usr/local/sbin/*
+
+    - name: 'Scaner: Print scan results'
```
Comment (MateuszGrabuszynski) [inline .github/workflows/ubuntu-build.yml:110]: [minor]
```suggestion
    - name: 'Scanner: Print scan results'
```

File: .github/workflows/dependency-review.yml [hunk 3]
```diff
@@ -1,21 +1,52 @@
...
+      with:
+        egress-policy: audit
+
+    - name: 'super-linter: checkout repository [fetch-depth=0]'
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      with:
+        fetch-depth: 0
+        ref: ${{ inputs.branch }}
+
+    - name: 'super-linter: perform supe-linter scan workflow.'
```
Comment (MateuszGrabuszynski) [inline .github/workflows/dependency-review.yml:49]:
```suggestion
    - name: 'super-linter: perform super-linter scan workflow.'
```

**Review by MateuszGrabuszynski — APPROVED** [review]
LGTM (marked a few typos, but non-blocking)

**Review by MateuszGrabuszynski — APPROVED** [review]
LGTM

**Pull request merged**